modules/consentManagement.js

```diff
@@ -182,6 +189,101 @@ function lookupIabConsent(cmpSuccess, cmpError, hookConfig) {
   }
 }
 
+function lookupUspConsent(uspSuccess, cmpError, hookConfig) {
+  function handleCmpResponseCallbacks() {
+    const uspResponse = {};
+
+    function afterEach() {
+      if (uspResponse.usPrivacy) {
+        uspSuccess(uspResponse, hookConfig);
+      }
+    }
+
+    return {
+      consentDataCallback: function (consentResponse) {
+        uspResponse.usPrivacy = consentResponse.usPrivacy;
```

Comment:
as per the related comments in this PR, the `uspapi` passes a second argument to the callback, which should be handled here or in the other function in the chain.